component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/twitter/TwitterAuthenticatorConstants.java

```diff
@@ -32,4 +32,5 @@
     public static final String TWITTER_OAUTH_VERIFIER = "oauth_verifier";
     public static final String TWITTER_CALLBACK_URL = "https://localhost:9443/commonauth";
     public static final String CLAIM_ID = "id";
+    public static final String CLAIM_DIALECT_URI_PARAMETER = "TwitterClaimDialectUri";
```

Comment:
This is configured inside an authenticator config right ? Shall we make this consistent ? The name should not have twitter, facebook or what the type of authenticator is. having ClaimDialect is enough. Shall we do this change in previous PRs as well ? Isn't this make sense ?

Reply:
Sure. I have fixed this as you requested.